wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -165,16 +186,16 @@ def save_pdf(self, response):
         # Retrieve metadata
         data_dict = response.meta.get('data_dict', {})
         # Download PDF file to /tmp
-        filename = response.url.split('/')[-1]
+        filename = urlparse(response.url).path.split('/')[-1]
```

Comment:
is there not a who_id in the document we can use? I assume that this strips out the params of a url?

Reply:
It isn't always present in the documents data, so at the moment, we still have to parse the url

Reply:
👍